src/AboutContent.js

```diff
@@ -9,9 +12,36 @@ const AboutContent = function ( props ) {
+      <Message>Please note that this app does not store user data, so <strong>if you refresh the page the data you've entered will be lost</strong>. Each time you go through the app, it's a clean slate.</Message>
+
+      <Header as='h3'>Who's doing this?</Header>
+      <p>This application is part of a project made possible by a Boston Foundation Open Door Grant to the University of Massachusetts Boston's <ExternalLink href='https://www.umb.edu/csp'>Center for Social Policy</ExternalLink>, in close partnership with <ExternalLink href='http://www.prohope.org/'>Project Hope</ExternalLink> and <ExternalLink href='http://www.codeforboston.org/'>Code for Boston</ExternalLink>. The Center for Social Policy is the lead partner for the <ExternalLink href='https://onsolidgroundma.org/'>On Solid Ground Coalition</ExternalLink>.</p>
+      <p>The code base is being maintained <ExternalLink href='https://github.com/codeforboston/cliff-effects'>on GitHub</ExternalLink> by <ExternalLink href='http://www.codeforboston.org/'>Code for Boston</ExternalLink> volunteers. For more information or to report a bug, please contact andrew@codeforboston.org.</p>
+
+      <Header as='h3'>Thank You</Header>
+      <p>Here's a special thank you to all the Code for Boston volunteers who brought you this application, especially Annie LaCourt, Isaac Chansky, Michelle Bernstein, Alec Danaher, Sasha Maryl, Drew Love, Liani Lye, Liam Morley, Nick Francisci, Stephen Chin, Shameek Poddar, Will McIntosh, Don Blair, Ethan Strominger, Nick Lee, Jonathan Marcus, and Emily Wasserman.</p>
+
+      <Header as='h3' name={'learn-more'}>Learn More</Header>
+      <p>A bit more about cliff effects:</p>
+      <ul>
+        <li><ExternalLink href='https://www.youtube.com/watch?v=BveX_rID4_E'>https://www.youtube.com/watch?v=BveX_rID4_E</ExternalLink></li>
+        <li><ExternalLink href='http://www.nccp.org/projects/files/NCCP_CO_presentation07.pdf'>http://www.nccp.org/projects/files/NCCP_CO_presentation07.pdf</ExternalLink></li>
+        <li><ExternalLink href='https://www.umb.edu/editor_uploads/images/centers_institutes/center_social_policy/The_Road_to_the_Cliff_Edge_08.16.17.pdf'>https://www.umb.edu/editor_uploads/images/centers_institutes/center_social_policy/The_Road_to_the_Cliff_Edge_08.16.17.pdf</ExternalLink></li>
+      </ul>
+
+      <br/>
+      <Message><strong>Important note:</strong> This application is a minimum viable product. It should not be used as the sole tool to understand a client's eligibility for SNAP and Section 8, or for any other public assistance program.</Message>
```

Comment:
Let's also pop this block to the top of the page, or at least to the 'What ...' section, which is where this is most pertinent.

Reply:
Maybe as a 'warning' component or some such as opposed to a 'message' one?